variants/PNUCLEO_WB55RG/otp.h

```diff
@@ -0,0 +1,53 @@
+/**
+  ******************************************************************************
+  * @file    otp.h
+  * @author  MCD Application Team
+  * @brief   OTP manager  interface
+  ******************************************************************************
+   * @attention
+  *
+  * <h2><center>&copy; Copyright (c) 2019 STMicroelectronics.
+  * All rights reserved.</center></h2>
+  *
+  * This software component is licensed by ST under BSD 3-Clause license,
+  * the "License"; You may not use this file except in compliance with the
+  * License. You may obtain a copy of the License at:
+  *                        opensource.org/licenses/BSD-3-Clause
+  *
+  ******************************************************************************
+ */
+
+
+/* Define to prevent recursive inclusion -------------------------------------*/
+#ifndef __OTP_H
+#define __OTP_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Includes ------------------------------------------------------------------*/
+/* ----------------------------------- *
+ *  Packed usage (compiler dependent)  *
+ * ----------------------------------- */
+#undef PACKED__
+#undef PACKED_STRUCT
+
```

Comment:
Useless can be removed

```suggestion
/* Exported types ------------------------------------------------------------*/
```